gems/pending/openstack/events/openstack_ceilometer_event_monitor.rb

```diff
@@ -9,8 +9,8 @@ def self.available?(options = {})
       return true
     rescue => ex
       $log.debug("Skipping Openstack Ceilometer events. Availability check failed with #{ex}.") if $log
+      raise ex
```

Comment:
Prefer `raise` which no argument if you are going to reraise the same exception.